sway/desktop/render.c

```diff
@@ -1722,29 +1837,42 @@ static void render_container(struct sway_output *output,
 		.children = con->current.children,
 		.focused = focused,
 		.active_child = con->current.focused_inactive_child,
+		.on_focused_workspace = is_focused_ws,
 	};
 	render_containers(output, damage, &data);
 }
 
 static void render_workspace(struct sway_output *output,
-		pixman_region32_t *damage, struct sway_workspace *ws, bool focused) {
-	struct parent_data data = {
-		.layout = ws->current.layout,
-		.box = {
-			.x = floor(ws->current.x),
-			.y = floor(ws->current.y),
-			.width = ws->current.width,
-			.height = ws->current.height,
-		},
-		.children = ws->current.tiling,
-		.focused = focused,
-		.active_child = ws->current.focused_inactive_child,
-	};
-	render_containers(output, damage, &data);
+		pixman_region32_t *damage, struct sway_workspace *ws, bool focused,
+		struct sway_workspace *other_ws) {
```

Comment:
I think `other_ws` could be more descriptive, something like `new_focused_ws` would be more descriptive